modules/hello-world/pages/sample-application.adoc

```diff
@@ -1,9 +1,180 @@
-= Sample App Tutorial
+= Sample Application
+:nav-title: Travel Sample App
 :content-type: tutorial
+:page-topic-type: tutorial
 :page-aliases: ROOT:sample-application,ROOT:tutorial4,ROOT:sample-app-backend
 
+[abstract]
+Discover how to program interactions with the Couchbase Server via the data, Query, and search services -- using the Travel Sample Application with the built-in Travel Sample data Bucket.
+This version also demonstrated the Developer Preview of the upcoming Collections feature.
 
-include::ref:partial$beta-warning.adoc[]
+6.5@sdk:pages:partial$sample-application.adoc
+
+This version of the Travel Sample App uses the xref:6.5@server:developer-preview:preview-mode.adoc[Collections Developer Preview (DP)] 
+-- enabling Developer Preview should only be done on a development machine; 
+there is no upgrade path available from a DP-enabled Couchbase Server.
+
+== Preparation
+
+As well as the xref:start-using-sdk.adoc[Java SDK 3.0] and Couchbase Server, 
+set up as described above, you will need `git` to fetch the travel sample application code:
+
+[source,bash]
+----
+git clone git@github.com:couchbaselabs/try-cb-java.git
+----
+
+Change directory into your cloned repository, and check out the latest branch (this will most probably be enabled as the default branch).
+
+[source,bash]
+----
+git checkout 6.5.0-branch
+----
+
+Before building the Travel Sample Application, which in the case of the Java SDK utilises the Collections _Developer Preview_, you need to enable this DP feature.
+Remember, do not run this on a production server: Collections is  _Developer Preview_, and is not supported for _production_ use with Couchbase Server 6.5; 
+there is no upgrade path from a DP-enabled server.
+
+[source,bash]
+.Enable Collections Developer Preview
+----
+/opt/couchbase/bin/couchbase-cli enable-developer-preview --enable -c http://localhost:8091 -u Administrator -p password
+----
++
+[source,bash]
+----
+Developer preview cannot be disabled once it is enabled. If you enter developer preview mode you will not be able to upgrade. DO NOT USE IN PRODUCTION.
```

Comment:
this is the third time it is mentioned in quick succession - maybe this one tip is enough?